### ISSUE TITLE
chart.update series mouseOver leaves old event handlers in place for points already mouseOvered

### ISSUE THREAD

**Issue by anajavi** [highcharts/highcharts#12067]
#### Expected behaviour
mouseOver eventhandler should be the same for all points after updating eventhandlers.

#### Actual behaviour
Old eventhandlers are left in place for those points which have been hovered.

Probably related to #6538, but this time some handlers are actually updated.

#### Live demo with steps to reproduce
https://jsfiddle.net/vopcu8mt/

Hover on one of points. Then press update. Now the point that was hovered has old event handler. Other points have the new handler

#### Product version
Highcharts: 7.2.0

#### Affected browser(s)
Chrome 77

#### Original problem
https://github.com/whawker/react-jsx-highcharts/issues/229

**Comment by pawelfus** [comment]
Thanks for reporting the issue @anajavi 

**Workaround:**
Instead of adding events just once, add them on every interaction (may decrease the performance!)
Demo: https://jsfiddle.net/BlackLabel/bd5tvsn9/

Plugin:
```js
(function(H) {
  H.Point.prototype.importEvents = function() {
    var point = this;

    if (this.importedEvents) {
      this.importedEvents.forEach(function(unbinder) {
        unbinder();
      });
    }

    this.importedEvents = [];

    var options = H.merge(
        point.series.options.point,
        point.options
      ),
      events = options.events;

    point.events = events;

    H.objectEach(events, function(event, eventType) {
      if (H.isFunction(event)) {
        point.importedEvents.push(
          H.addEvent(point, eventType, event)
        );
      }
    });
  }
})(Highcharts);
```


*Internal note:*
This is caused by `importEvents()` - which is called **only** on the first interaction with a point. Once events are imported, updating events won't do anything until point is destroyed. This is related to performance: no need to apply events to all points on init - with 1k points we would create 1k of event handlers while a user may not even mouseover the chart.

**Comment by anajavi** [comment]
I think this should be still kept open. It makes usage through react libraries quite complex.

**Comment by anajavi** [comment]
seems to be fixed in highcharts 11.4
